backends/gstreamer/player.rs

```diff
@@ -418,19 +418,18 @@ impl GStreamerPlayer {
         }
 
         let player = gst_player::Player::new(
-            /* video renderer */ None, /* signal dispatcher */ None,
+            None::<gst_player::PlayerVideoRenderer>,
+            None::<gst_player::PlayerSignalDispatcher>,
```

Comment:
`gst_player::Player::default()` but ideally (as you require 1.22 here now, don't you?) you could use `gst_play` instead. But that should probably happen in a separate PR.